src/commands/release/import.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import type { ReadStream } from 'fs';
+import { promises as fs } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an app or fleet.
+
+		Imports a release from a file to an app or fleet. The revision field of the release
+		is automatically omitted when importing a release. The backend will auto-increment
+		the revision field of the imported release if a release exists with the same semver.
+		A release will not be imported if a successful release with the same commit already
```

Comment:
Unrelated to the PR itself but I'm curious why is this the case? What were the downsides of assigning a new commit to the release if the same commit already existed? (in which case we would output the new commit as well at the end of this command)

Reply:
We want to emphasize that the imported release is an exact copy of another release.  Since users are allowed to override the version of an imported release, the commit helps confirm if the release is a copy of a specific build from another fleet.  We also do not get any real advantage for allowing imports of the same copy of the release if a successful release with the same commit already exists.